hack/version.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Copyright 2014 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# (Loosely) used https://github.com/kubevirt/kubevirt/blob/master/hack/version.sh as a template
```

Comment:
If loosely, we could add another line to Copyright section

Reply:
Which one? I actually used the line from the original; I haven't added one for this patch (because AFAIK copyright laws don't require authorship being added at all).

Reply:
fair enough